### PR TITLE
Add Cart Item stories

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- Add stories for Cart Item component based on the UI kit

- Remove stories for more specific "Cart Entry Item" component

## Justify why they are needed

- Focusing Storybook on being a tool to develop UI components

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2753** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2753" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2753?utm_source=stack-comment).

**Review by guilhermespopolin — APPROVED** [review]

File: apps/store/src/components/CartItem/CartItem.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import { Button, theme } from 'ui'
+import { CurrencyCode, InsuranceDocumentType } from '@/services/apollo/generated'
+import { CartItem } from './CartItem'
+
+const meta: Meta<typeof CartItem> = {
+  title: 'Components / Cart Item',
+  component: CartItem,
+}
+
+export const ReadOnly: StoryObj<typeof CartItem> = {
```
Comment (gustaveen) [inline apps/store/src/components/CartItem/CartItem.stories.tsx:11]: nit:
```suggestion
type Story = StoryObj<typeof CartItem>

export const ReadOnly: Story = {
```

**Review by gustaveen — APPROVED** [review]
Very nice! ⭐️

**Pull request merged**